azerothcore/azerothcore-wotlk
# feat(Core/Player): Stats limitation

* cherry-pick commit (https://github.com/TrinityCore/TrinityCore/commit/e18571346400281861490b36c4d35a381bc80dd8)

* closes https://github.com/azerothcore/azerothcore-wotlk/issues/3648

Co-Authored-By: Corbi <12052933+corbi@users.noreply.github.com>



## Changes Proposed:
-  Add option to choose stat limitation if one wants
- Disabled by default

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3648

## SOURCE:


## Tests Performed:

- None

## How to Test the Changes:


1. Enable the config
2. Set the limitation % to what you like
3. See that Crit/Dodge/Parry/Block percentages cannot exceed the limitation


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
